django_filters/rest_framework/filterset.py

```diff
@@ -36,3 +37,12 @@ def __init__(self, *args, **kwargs):
             helper.layout = Layout(*layout_components)
 
             self.form.helper = helper
+
+    @property
+    def qs(self):
+        from rest_framework.exceptions import ValidationError
+
+        try:
+            return super(FilterSet, self).qs
+        except forms.ValidationError as e:
+            raise ValidationError(utils.raw_validation(e))
```

Comment:
@tomchristie Would you be interested in pulling the `utils.raw_validation` helper here into DRF? (This mapping of `django.forms.ValidationError` to `rest_framework.exceptions.ValidationError` is a fairly common issue.)